packages/deployment/app/cloudformation.template.yml

```diff
@@ -126,6 +126,24 @@ Resources:
   # SNS config BEGIN
   #################################################
 {{#each sns}}
+  {{#if this.arn}}
```

Comment:
Not sure why you have this `if` here. It looks like other than `DisplayName` both sides of the if statement are identical. If that is so, you should put the if around the `DisplayName` instead of the whole block.

Reply:
You're right. I had less stuff originally, but then I had to add stuff back in. I think the display name might be fine to have anyway. I'll test it out.